packages/block-library/src/quote/index.js

```diff
@@ -36,9 +36,13 @@ export const name = 'core/quote';
 
 export const settings = {
 	title: __( 'Quote' ),
-	description: __( 'Maybe someone else said it better -- add some quoted text.' ),
+
+	description: __( 'Give quoted text visual emphasis. "In quoting others, we cite ourselves." — Julio Cortázar' ),
```

Comment:
Other user-facing strings use fancy quotes instead of typewriter quotes, e.g. the NUX tips.